model/naive_users.go

```diff
@@ -72,6 +73,9 @@ func (*NaiveUserManager) IsRedirect() bool                          { return fal
 func (um *NaiveUserManager) GetUserByID(id string) (gimlet.User, error) {
 	for _, user := range um.users {
 		if user.User == id {
+			if user.Invalid {
```

Comment:
We've added a native (basic users and basic user manager) to the gimlet package. It might be the case that we can get rid of this implementation and just use that.

Reply:
Unfortunately, the basic user and user manager's GetUserById function do not support invalid users.

Reply:
can we add that functionality there.